src/scratchtocatrobat/converter/converter.py

```diff
@@ -592,6 +600,46 @@ def converted_project_for(cls, scratch_project, progress_bar=None, context=None)
         assert catrobat.is_background_sprite(catrobat_project.getDefaultScene().getSpriteList().get(0))
         return ConvertedProject(catrobat_project, scratch_project)
 
+    def _add_bricks_for_visible_variables(self, catrobat_scene):
+        scratch_project = self.scratch_project
+        brick_dict = dict()
+        sprite_dict = dict()
+        sprite_list = catrobat_scene.getSpriteList()
+        for sprite in sprite_list:
+            sprite_name = sprite.getName()
+            # TODO: rewrite when there is a general name for the background sprite in the code
+            if sprite_name == "Hintergrund":
+                sprite_name = "Stage"
+            sprite_dict[sprite_name] = sprite
+        for var in scratch_project._var_to_visibility_map:
+            if(scratch_project._var_to_visibility_map.get(var)[0]):
+                target_sprite = scratch_project._var_to_visibility_map.get(var)[1]
```

Comment:
Use more descriptive variable name (see comment below in corresponding test function).